packages/react-native-reanimated/src/createAnimatedComponent/PropsFilter.tsx

```diff
@@ -67,16 +67,32 @@ export class PropsFilter implements IPropsFilter {
         has('workletEventHandler', value) &&
         value.workletEventHandler instanceof WorkletEventHandler
       ) {
-        if (value.workletEventHandler.eventNames.length > 0) {
-          value.workletEventHandler.eventNames.forEach((eventName) => {
-            props[eventName] = has('listeners', value.workletEventHandler)
-              ? (
-                  value.workletEventHandler.listeners as Record<string, unknown>
-                )[eventName]
-              : dummyListener;
-          });
+        const handler = value.workletEventHandler;
+        const isWebHandler = has('listeners', handler);
+        const hasJSHandlers = Object.keys(handler.JSHandlers).length > 0;
+
+        if (hasJSHandlers) {
```

Comment:
Does this mean that a worklet event handler can also be a JS event handler? 🤔

Reply:
Nope, they are seperate